### PR TITLE
Implement finding an edit part in other edit part

### PR DESCRIPTION
**Pull request by apodhrad**
This will require implementing of WithParent matcher, for an example have a look at Activity implementation ib BPEL tests.

### CODE REVIEW

File: plugins/org.jboss.reddeer.gef/src/org/jboss/reddeer/gef/matcher/IsEditPartWithParent.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+
+/**
+ * Returns true if an edit part is a child of a given parent.
+ * 
+ * @author Andrej Podhradsky (apodhrad@redhat.com)
+ *
+ */
+public class IsEditPartWithParent extends BaseMatcher<EditPart> {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.gef/src/org/jboss/reddeer/gef/matcher/IsEditPartWithParent.java]: shouldnt this be called EditPartWithParent (without "Is") ? Seems to be more aligned with names of other matches

Comment (apodhrad) [inline plugins/org.jboss.reddeer.gef/src/org/jboss/reddeer/gef/matcher/IsEditPartWithParent.java]: I dno't see any problem with this naming convention... there are matchers such as WithLabel IsInstanceOf, so why not to use IsEditPartWithParent? For me it is very meaningful.

Comment (rawagner) [inline plugins/org.jboss.reddeer.gef/src/org/jboss/reddeer/gef/matcher/IsEditPartWithParent.java]: allright :)

**Comment by rawagner** [comment]
pushed to master

**Pull request closed without merging**